### ISSUE TITLE
Disallow user to delete record if it has  any type of comments

### ISSUE THREAD

**Issue by jimmrom** [Serhioromano/cobalt-9#38]
I think we all agree that a user should not be able to delete a record which received comments,example:

user1 has a record which received 5 comments from other users
user1 should not be able to delete that record if it received comments
he should be able to delete the records only if there are no comments ( or at last an option where you can decide that )

**Comment by matijaveber** [comment]
I disagree. If you enable for users to add records then my opinion is they need to be in full control of what they publish. If they decide to remove record (with comments or not) why not. That's like to not be able to remove facebook posts, tweets,... if they have comments. It has to be my choice if it's my content.

**Comment by Serhioromano** [comment]
@vebrovski it might have a sense in some cases. For instance support desk. The user creates a support request and if a conversation is started he cannot delete it. Later he may complain that service was not good or something. And request has to be in the system. But if the conversation has not been started, the user may still delete request.
